apps/web/app/features/stacking/hooks/use-user-positions-fake-loading.ts

```diff
@@ -0,0 +1,15 @@
+import { useEffect } from 'react';
+
+import { useFakeLoading } from '~/utils/fake-loading';
+
+export function useUserPositionsFakeLoading(isLoading: boolean) {
```

Comment:
Skeleton loading?

Reply:
the sense of it is fake loading, not skeleton